new-dti-website-redesign/src/components/TimelineCard.tsx

```diff
@@ -0,0 +1,223 @@
+import Link from 'next/link';
+import IconWrapper from './IconWrapper';
+
+export type DateTime = {
+  date: string;
+  isTentative: boolean;
+  time?: string;
+};
+
+export type RecruitmentEvent = {
+  title: string;
+  description: string;
+  location?: string;
+  type: string;
+  link?: string;
+  freshmen: DateTime;
+  upperclassmen: DateTime;
+  spring: DateTime;
+};
+
+const TimelineCard = ({
+  event,
+  cycle
+}: {
+  event: RecruitmentEvent;
+  cycle: 'freshmen' | 'upperclassmen' | 'spring';
+}) => {
+  const datetime: DateTime = event[cycle];
+  return (
+    <div className="max-w-[504px]">
+      <div className="flex flex-col gap-2 p-4 rounded-t-lg border border-border-1 border-b-transparent">
+        <IconWrapper size="small">
+          <svg
+            xmlns="http://www.w3.org/2000/svg"
+            width="24"
+            height="24"
+            viewBox="0 0 24 24"
+            fill="none"
+          >
+            <path
+              d="M12 22C17.5228 22 22 17.5228 22 12C22 6.47715 17.5228 2 12 2C6.47715 2 2 6.47715 2 12C2 17.5228 6.47715 22 12 22Z"
+              stroke="white"
+              stroke-width="2"
+              stroke-linecap="round"
+              stroke-linejoin="round"
+            />
+            <path
+              d="M12 16V12"
+              stroke="white"
+              stroke-width="2"
+              stroke-linecap="round"
+              stroke-linejoin="round"
+            />
+            <path
+              d="M12 8H12.01"
+              stroke="white"
+              stroke-width="2"
+              stroke-linecap="round"
+              stroke-linejoin="round"
+            />
+          </svg>
+        </IconWrapper>
+        <div className="flex flex-col gap-1">
+          <h5>{event.title}</h5>
+          <p className="text-foreground-3">{event.description}</p>
+        </div>
+      </div>
+      <div className="flex flex-col md:flex-row">
+        {event.location &&
+          (event.link ? (
+            <Link
+              href={event.link}
+              target="_blank"
+              className="flex justify-between md:w-1/2 p-4 md:rounded-bl-lg border border-border-1 hover:bg-background-2 underline underline-offset-3"
```

Comment:
Also adding the `focusState` class for the customs styled focus states for consistency

```suggestion
              className="flex justify-between md:w-1/2 p-4 md:rounded-bl-lg border border-border-1 hover:bg-background-2 underline underline-offset-3 focusState"
```

|Before|After|
|-|-|
|<img width="541" alt="Screenshot 2025-05-13 at 7 46 02 PM" src="https://github.com/user-attachments/assets/f0027574-ac71-4c85-85e4-c156538988c2" />|<img width="541" alt="Screenshot 2025-05-13 at 7 45 55 PM" src="https://github.com/user-attachments/assets/534fb45e-5509-449e-9a41-45fdef2570f0" />|